app/views/layouts/application.html.erb

```diff
@@ -66,6 +66,12 @@
       <% if flash[:info] %>
         <p><%= flash[:info] %></p>
       <% end %>
+        <div id="flashes">
+          <% flash.each do |name, msg| %>
+          <%= content_tag :div, raw(msg), id: "flash_#{name}",
```

Comment:
I'm wary of this at the moment as I just had a big todo about not letting scoreboards be raw.  Can we at least use 'sanitize' (http://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize) instead? That will at least strip out script tags.